### PR TITLE
cars page is done

### PR DESCRIPTION
**Pull request by nidhi0003**

### CODE REVIEW

File: src/app/pages/cars/cars.component.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<app-card [cardDetails]="cardDetails" [cards]="carDetails"></app-card>
+<h1>{{carDetails}}</h1>
```
Comment (Chiru003) [inline src/app/pages/cars/cars.component.html:2]: remove this

File: src/app/services/cars/cars.service.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import { Injectable } from '@angular/core';
+import { carDetail } from '../../types/cars';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CarsService {
+
+  constructor() { }
+  carDetails: carDetail [] = [
```
Comment (nidhi0003) [inline src/app/services/cars/cars.service.ts:10]: remove the space

Comment (Chiru003) [inline src/app/services/cars/cars.service.ts:10]: remove the space

Comment (nidhi0003) [reply src/app/services/cars/cars.service.ts:10]: space is removed

File: src/app/pages/cars/cars.component.html (hunk 1 above)
Comment (nidhi0003) [reply src/app/pages/cars/cars.component.html:2]: I have removed <h1> tag

File: src/app/types/cars.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export interface carDetail {
+    name:string,
+    brand:string,
+    fuel:string,
+    price:number,
+    coverFileName:string;
+    description:string,
+}
```
Comment (nidhi0003) [inline src/app/types/cars.ts:8]: Here removed ; semicolon

**Review by Chiru003 — APPROVED** [review]

**Pull request merged**